example/reader_util.c

```diff
@@ -1700,11 +1694,15 @@ struct ndpi_proto ndpi_workflow_process_packet(struct ndpi_workflow * workflow,
     iph6 = (struct ndpi_ipv6hdr *)&packet[ip_offset];
     proto = iph6->ip6_hdr.ip6_un1_nxt;
     ip_len = ntohs(iph6->ip6_hdr.ip6_un1_plen);
+    if (header->caplen < ip_offset + sizeof(struct ndpi_ipv6hdr + ntohs(iph_v6->ip6_hdr.ip6_un1_plen))
```

Comment:
I had to fix it as this line contains a syntax error. Please check your code before pulling it

Reply:
Sorry, I thought I had checked but I must have checked before adding the syntax error